app/controllers/events_controller.rb

```diff
@@ -293,7 +293,7 @@ def add_event_query_conditions(query)
 
   def filter_application_letters(application_letters)
     application_letters = application_letters.to_a
-    filters = (params[:filter] || {}).select { |_k, v| v == '1' }.map { |k, _v| k.to_s }
+    filters = (params[:filter] || {}).select { |_k, v| v == '1' }.each { |k, _v| k.to_s }
```

Comment:
If I understand this correctly, `each` modifies the original sequence, whereas `map` returns a new sequence. In this case the assignment is confusing and doesn't make sense? `Map` doesn't work in this case, because this method is not implemented for [ActionController::Parameters](http://api.rubyonrails.org/classes/ActionController/Parameters.html). In earlier Rails versions,  `params` was a Hash (where 'map' is implemented), but now it is a separate object.

Reply:
I got a little bit confused. The `each` indeed doesn't modify the underlying array. In this case the `transform_value`method can be used. In the user case I have used `to_unsafe_h` for now. Thus it should behave like before the update.